ui/src/containers/CreateVolume.js

```diff
@@ -150,12 +157,6 @@ const LabelsName = styled(LabelsValue)`
   color: ${(props) => props.theme.brand.textPrimary};
 `;
 
-const PageContainer = styled.div`
-  display: flex;
-  // To avoid the big empty space on the right side of page
-  justify-content: space-around;
-`;
-
```

Comment:
We should keep this styling to put this creation form in the middle of the page to avoid the big empty space on the right side of the page.

Reply:
Oh ok. On the design it was aligned left : 
![Screenshot 2021-03-16 at 15 37 35](https://user-images.githubusercontent.com/3480526/111327325-95efa580-866d-11eb-812c-2610fb5578fb.png)

Will move it back to the center.

Reply:
@alexis-ld 
thank you! this should be the correct one!
![image](https://user-images.githubusercontent.com/18453133/111436970-dd744100-8702-11eb-91df-f1c8b2b8bf62.png)